ArmPkg/Drivers/CpuDxe/CpuMmuCommon.c

```diff
@@ -90,6 +90,7 @@ SetGcdMemorySpaceAttributes (
   UINTN                 EndIndex;
   EFI_PHYSICAL_ADDRESS  RegionStart;
   UINT64                RegionLength;
+  UINT64                Capabilities; // MU_CHANGE: Sync AARCH64 GCD Capabilities with Page Table

```

Comment:
Leaving a comment here on design decision. I think this should be refactored into a separate function, a la `RefreshGcdMemoryAttributesFromPaging` to make this a one time syncing, not on each call to SetGcdMemorySpaceAttributes.

Reply:
Per offline discussion, I decided to leave this functionality in this function, as it will sync the GCD's capabilities whenever the attributes are set, which follows the x64 pattern.